src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -265,7 +266,7 @@ public JSONObject getBlockChainInfo() throws org.json.simple.parser.ParseExcepti
       System.out.println(url.toString());
       HttpURLConnection con = (HttpURLConnection) url.openConnection();
       String userPassword = BITCOIN_NODE_USERNAME + ":" + BITCOIN_NODE_PASSWORD;
-      String encoding = new sun.misc.BASE64Encoder().encode(userPassword.getBytes());
+      String encoding = java.util.Base64.getEncoder().encodeToString(userPassword.getBytes());
```

Comment:
Why do use the fully qualified version here. java.util can be imported normally.